### PR TITLE
Fixed refine_edges keyword to allow it to be referenced in Python

### PR DESCRIPTION
**Pull request by DanielChaseButterfield**
This is to solve Issue #243. The functionality was already there, but "refine-edges" can't be used as a keyword argument in python because of the '-'.

### CODE REVIEW

**Comment by christian-rauch** [comment]
In the end, this was just a typo? Can you maybe rephrase the commit message saying that this fixes a typo in the `apriltag_new` parameters?

**Comment by DanielChaseButterfield** [comment]
@christian-rauch Sure!

**Comment by christian-rauch** [comment]
@DanielChaseButterfield It now looks like you have two commits applying the same change and an additional merge commit. Can you rebase on `master` and squash such that this one-line change is in a single commit on this PR?

**Comment by DanielChaseButterfield** [comment]
Okay, I think it worked now :)

**Review by wxmerkt — APPROVED** [review]
